applications/openshift/high-availability/three_control_plane_nodes/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+title: 'Ensure machine count of MachineConfigPool master is 3'
+
+description: |- 
+    To ensure, that the OpenShift control plane stays accessible on outage of a single master node, a 
+    number of 3 control plane nodes is required.
+
+rationale: |-
+    A highly-available OpenShift control plane requires 3 control plane nodes. This allows etcd to have
+    a functional quorum state, when a single control plane node is unavailable.
+
+identifiers: {}
```

Comment:
unique identifier is missing, add it by using 
`./utils/rule_dir_json.py` followed by `python utils/fix_rules.py --product products/ocp4/product.yml add-cce --cce-pool redhatthree_control_plane_nodes`